doc/source/user_guide/scale.rst

```diff
@@ -374,5 +374,101 @@ datasets.
+along both columns and rows because it gives Modin flexibility and scalability in both the number of columns and
+the number of rows. Let's take a look at how we can read the data from a CSV file with Modin the same way as with pandas
+and perform a simple operation on the data.
+
+.. ipython:: python
+
+   import pandas
+   import modin.pandas as pd
+   import numpy as np
+
+   array = np.random.randint(low=0.1, high=1.0, size=(2 ** 20, 2 ** 8))
+   filename = "example.csv"
+   np.savetxt(filename, array, delimiter=",")
+
+   %time pandas_df = pandas.read_csv(filename, names=[f"col{i}" for i in range(2 ** 8)])
+   CPU times: user 48.3 s, sys: 4.23 s, total: 52.5 s
+   Wall time: 52.5 s
+   %time pandas_df = pandas_df.map(lambda x: x + 0.01)
+   CPU times: user 48.7 s, sys: 7.8 s, total: 56.5 s
+   Wall time: 56.5 s
+
+   %time modin_df = pd.read_csv(filename, names=[f"col{i}" for i in range(2 ** 8)])
+   CPU times: user 9.49 s, sys: 2.72 s, total: 12.2 s
+   Wall time: 17.5 s
+   %time modin_df = modin_df.map(lambda x: x + 0.01)
+   CPU times: user 5.74 s, sys: 1e+03 ms, total: 6.74 s
+   Wall time: 2.54 s
+
+We can see that Modin has been able to perform the operations much faster than pandas due to distributing execution.
+Even though Modin aims to speed up each single pandas operation, there are the cases when pandas outperforms.
+It might be a case if the data size is relatively small or Modin hasn't implemented yet a certain operation
+in an efficient way. Also, for-loops is an antipattern for Modin since Modin has initially been designed to efficiently handle
+heavy tasks, rather a small number of small ones. Yet, Modin is actively working on eliminating all these drawbacks.
+What you can do for now in such a case is to use pandas for the cases where it is more beneficial than Modin.
+
+.. ipython:: python
+
+   from modin.pandas.io import to_pandas
+
+   %%time
+   pandas_subset = pandas_df.iloc[:100000]
+   for col in pandas_subset.columns:
+      pandas_subset[col] = pandas_subset[col] / pandas_subset[col].sum()
+   CPU times: user 210 ms, sys: 84.4 ms, total: 294 ms
+   Wall time: 293 ms
+
+   %%time
+   modin_subset = modin_df.iloc[:100000]
+   for col in modin_subset.columns:
+      modin_subset[col] = modin_subset[col] / modin_subset[col].sum()
+   CPU times: user 18.2 s, sys: 2.35 s, total: 20.5 s
+   Wall time: 20.9 s
+
+   %%time
+   pandas_subset = to_pandas(modin_df.iloc[:100000])
+   for col in pandas_subset.columns:
+      pandas_subset[col] = pandas_subset[col] / pandas_subset[col].sum()
+   CPU times: user 566 ms, sys: 279 ms, total: 845 ms
+   Wall time: 731 ms
+
+You could also rewrite this code a bit to get the same result with much less execution time.
+
+.. ipython:: python
+
+   %%time
+   modin_subset = modin_df.iloc[:100000]
+   modin_subset = modin_subset / modin_subset.sum(axis=0)
+   CPU times: user 531 ms, sys: 134 ms, total: 666 ms
+   Wall time: 374 ms
```

Comment:
This looks like it could be misleading, but is in fact not. It seems to me a reader might think that the same change to the pandas code would also decrease its execution time but pandas remains roughly the same. I think it'd be clearer to add timing for the pandas equivalent here too:

```
pandas_subset = pandas_df.iloc[:100000]
pandas_subset = pandas_subset / pandas_subset.sum()
```

Reply:
Added, thanks!

Reply:
I'm not seeing the same thing on my machine.

L438: You report 293ms, I get 52ms
L462: You report 72.2ms, I get 69ms

Can you check your timings again? If they hold up, what version of pandas/NumPy are you running?

Reply:
The time might be different for my and your runs. I have Intel(R) Xeon(R) Platinum 8468. Reran again and updated the timings. I have these pandas and numpy versions installed.

```bash
pandas                    2.2.0
numpy                     1.26.0
```